lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -143,13 +144,21 @@
     required: false
     default: 0
     description:
-    - Service memory limit in MB. 0 equals no limit.
+    - "Service memory limit (format: C(<number>[<unit>])). Number is a positive integer.
+      Unit can be C(B) (byte), C(K) (kibibyte, 1024B), C(M) (mebibyte), C(G) (gibibyte),
+      C(T) (tebibyte), or C(P) (pebibyte). Minimum is C(4M)."
```

Comment:
Where did you got the information from about `Minimum is C(4M).`?

Reply:
I assumed that the minimum was the same as described here https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory

Reply:
Hmm, that could well be. It might be better to leave it away when we're not sure, though; if someone puts an invalid value here, Docker will complain anyway.